features/ui/button/customButton.tsx

```diff
@@ -142,10 +148,159 @@ const ButtonContainer = styled(Button)<{
         return css`
           background: transparent;
           border: none;
-          color: ${props.isDisabled
-            ? color("gray", 300)
+          color: ${props.disabled ? color("gray", 300) : color("primary", 700)};
+
+          &:hover {
+            background: ${color("primary", 50)};
+          }
+
+          &:focus {
+            background: transparent;
+          }
+        `;
+      case ButtonColor.emptyGray:
+        return css`
+          background: transparent;
+          border: none;
+          color: ${props.disabled ? color("gray", 300) : color("gray", 500)};
+
+          &:hover {
+            background: ${color("gray", 50)};
+          }
+
+          &:focus {
+            background: transparent;
+          }
+        `;
+      case ButtonColor.error:
+        return css`
+          background: ${props.disabled
+            ? color("error", 200)
+            : color("error", 600)};
+          border: 1px solid
+            ${props.disabled ? color("error", 200) : color("error", 600)};
+          color: #fff;
+
+          &:hover {
+            background: ${color("error", 700)};
+            border-color: ${color("error", 700)};
+          }
+
+          &:focus {
+            background: ${color("error", 600)};
+            border-color: ${color("error", 600)};
+          }
+        `;
+    }
+  }}
+`;
+
+const StyledLink = styled(Link)<{
```

Comment:
it seems to have started when I made this styled component because eslint when committing was giving an overload error for href when I just nested it in the return component.